docs/docs/dev_docs/getting_started/sandbox.md

```diff
@@ -60,18 +62,27 @@ With the help of Aztec.js you will be able to:
 
 ## I have the Sandbox running, show me how to use it!
 
-Let's have a complete walkthrough from start to finish. I'm using WSL2 Ubuntu under Windows but the following should work under regular Linux or MacOS. We will deploy and use a private token contract on our Sandbox. Writing the contract itself is out of scope for this tutorial, we will use the Private Token Contract supplied as one of the example contracts.
+We will deploy and use a private token contract on our Sandbox. Writing the contract itself is out of scope for this tutorial, we will use the Private Token Contract supplied as one of the example contracts.
+
+The following should work for MacOS, Linux or even WSL2 Ubuntu under Windows. 
 
-Let's create an empty project called `private-token`. If you are familiar with setting up Typescript projects then you can fast-forward the next couple of steps.
+Let's create an empty project called `private-token`. If you are familiar with setting up Typescript projects then you can skip to step 6.
 
-Also, although I am using `yarn`, vanilla `npm` would work just as well.
+Although both `yarn` and `npm` would work, this example uses `yarn`. Open the terminal and do the following
 
+1. Ensure node version is 18 or more by running
+```sh
+node -v 
 ```
-~$ node -v
-v18.8.0
+
+2. Initialize a yarn project
+```sh
 ~$ mkdir private-token
 ~$ cd private-token
 ~/private-token$ yarn init
```

Comment:
See here if you wanted to do tabs for commands for each of npm or yarn. :) https://docusaurus.io/docs/markdown-features/code-blocks#multi-language-support-code-blocks